src/Commands/RunScan.ts

```diff
@@ -78,12 +76,11 @@ export class RunScan implements CommandModule {
           'A list of specific urls that should be included into crawler.'
       })
       .option('test', {
-        choices: Helpers.toArray(TestType),
```

Comment:
How can user see the full list of available tests? It seems we need to add a link to the list as well https://docs.brightsec.com/docs/vulnerability-guide (yet this doc does not point to the test names)

Reply:
update the help message and added link to documentation. We also update the docs, it contains test keywords now.